Reformatted Data-Dumper's Changes file as per CPAN::Changes::Spec

neilb opened the pull request:
This means that tools can parse this, and in particular MetaCPAN
will list the changes when you look at the dist/release page for
Data-Dumper.

neilb committed:
Reformatted Data-Dumper Changes as per CPAN::Changes::Spec

This means that tools can parse this, and in particular MetaCPAN
will list the changes when you look at the dist/release page for
Data-Dumper.

dist/Data-Dumpers/Changes used to be in pod, with a known pod
issue, so that's now dropped.

Updated AUTHORS and checkAUTHORS, as my github account has
an email address not currently known, but it's my prime
email address for Perl stuff.

Grinnz commented:
podcheck failure does not say what failed, strangely. The other failure is missing AUTHORS entry

Grinnz commented:
I suspect this file needs to be removed from known_pod_issues: https://github.com/Perl/perl5/blob/blead/t/porting/known_pod_issues.dat#L372

neilb commented:
Updated, thanks:

* updated AUTHORS to have my new email address as primary
* updated Porting/checkAUTHORS.pl so the email address map in __DATA__ knows my previous email address
* deleted the entry for Changes in t/porting/known_pod_issues.dat

nwc10 commented:
I cherry-picked this onto my branch https://github.com/nwc10/perl5/tree/Data-Dumper-2.181

It conflicted with my most recent Changes entry, so I fixed that. I also fixed the cosmetic missing newlines that we both spotted, and removed some trailing whitespace that `git diff --color` is very "in-your-face" about.

I'm assuming that this is good (in that, I'm going to build a tarball, test that, if good ship it, and merge my branch into blead)

nwc10 commented:
Pushed to blead and shipped to CPAN as 2.182
